supertokens_python/recipe/thirdpartyemailpassword/email_delivery/services/smtp/__init__.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from typing import Any, Dict
+
+from supertokens_python.ingredients.emaildelivery.service.smtp import \
+    EmailDeliverySMTPConfig
+from supertokens_python.ingredients.emaildelivery.types import \
+    EmailDeliveryInterface
+from supertokens_python.recipe.emailpassword.emaildelivery.service.smtp import \
+    SMTPService as EmailPasswordSMTPService
+from supertokens_python.recipe.emailpassword.types import \
+    TypeEmailPasswordEmailDeliveryInput
+from supertokens_python.recipe.thirdpartyemailpassword.types import \
+    TypeThirdPartyEmailPasswordEmailDeliveryInput
+
+
+class SMTPService(EmailDeliveryInterface[TypeThirdPartyEmailPasswordEmailDeliveryInput]):
+    ep_smtp_service: EmailPasswordSMTPService
+
+    def __init__(self, config: EmailDeliverySMTPConfig[TypeThirdPartyEmailPasswordEmailDeliveryInput]) -> None:
+        self.config = config
```

Comment:
No need to make into instance var

Reply:
Removed.

Reply:
This is still not done. Not sure why you commented removed.